python/examples/01-first-agent/main.py

```diff
@@ -5,7 +5,7 @@
 
 @agent.on_event("startup")
 async def introduce_agent(ctx: Context):
-    ctx.logger.info(f"Hello, I'm agent {ctx.name} and my address is {ctx.address}.")
+    ctx.logger.info(f"Hello, I'm agent {ctx.name} and my address is {ctx.identifier}.")
```

Comment:
I guess we should be consistent with what we call this in the logs, so either print the address here (I think that's slightly better) or call this the identity in the log statement.